src/app/shared/search/search-result.component.ts

```diff
@@ -128,7 +128,9 @@ export class SearchResultComponent implements OnChanges {
   }
 
   sort(column: SearchResultViewColumn) {
```

Comment:
where is the test for the case when the 
```this.comparator(column) === undefined```
?

Reply:
Done.